tests/cc/test_c_api.cc

```diff
@@ -0,0 +1,95 @@
+#include <stdint.h>
```

Comment:
sorry to be picky, but please add the license banner.
it matters for big companies adopting the software.

Disappointing that sput didn't last a week.
What are the chances that 'catch' will last longer?
I think it makes sense to move to gtest and stick with it.